app/(backend)/factories/serviceFactory.ts

```
@@ -0,0 +1,83 @@
+    return new ProviderService(
+      providerRepo, 
+      modelService,
+      apiKeyService
+    );
+  }
+  
+  /**
+   * Reset all service instances (useful for testing)
+   */
+  static resetServices(): void {
+    this.providerService = null;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)